### PR TITLE
WebAR-Chacmool: Ancient artifacts at Your Fingertips

### PR DESCRIPTION
**Pull request by jpmedley**
**Target Live Date:** 2018-06-13

- [x] This has been reviewed and approved by (John Pallet)
- [x] I have run `gulp test` locally and all tests pass.
- [x] I have added the appropriate `type-something` label.
- [ ] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

File: src/content/en/updates/2018/06/webar-chacmool.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+This demo was built over the course of about a month to build, leveraging some
+of the components from the WebXR team's first web based AR demo,
+[WebAR-Article](https://blog.google/products/google-vr/augmented-reality-web-everyone/).
+Information about the sculpture was sourced from its
+[Google's Arts & Culture](https://artsandculture.google.com/asset/a-chacmool-figure-holding-a-bowl-in-the-ruins-of-templo-mayor-in-mexico-city/nQFDnZcHsxuiiA)
+page, and the 3D model was provided by Google Arts & Culture's partner,
+[CyArk](https://artsandculture.google.com/partner/cyark).  To get the 3D model
+ready for the web, a combination of Meshlab, and Mesh Mixer was used to repair
+the model and decimate its mesh to decrease its file size. Then
+[Draco, a library for compressing and decompressing 3D geometric meshes and point clouds](https://github.com/google/draco)
```
Comment (petele) [inline src/content/en/updates/2018/06/webar-chacmool.md]: Should be: `[Draco](https://github.com/google/draco), a library for compressing and decompressing 3D geometric meshes and point clouds`

File: src/content/en/updates/2018/06/webar-chacmool.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Chacmool, an educational web demo, shows how easily web based AR can build an engaging experience.
+
+{# wf_updated_on: 2018-06-14 #}
+{# wf_published_on: 2018-06-14 #}
+{# wf_tags: immersive-web,webar,webxr #}
+{# wf_featured_image: /web/updates/images/2018/06/webar-chacmool.jpg #}
+{# wf_featured_snippet: Chacmool, an educational web demo, shows how easily web based AR can build an engaging experience. #}
+{# wf_blink_components: Blink>WebVR #}
```
Comment (petele) [inline src/content/en/updates/2018/06/webar-chacmool.md]: Should this be WebAR?

Comment (jpmedley) [reply src/content/en/updates/2018/06/webar-chacmool.md]: No.

Comment (petele) [reply src/content/en/updates/2018/06/webar-chacmool.md]: Hmm, OK, but it was added in `commonTags.json` below.

Comment (jpmedley) [reply src/content/en/updates/2018/06/webar-chacmool.md]: A subject index is different from a list of blink components.

Comment (jpmedley) [reply src/content/en/updates/2018/06/webar-chacmool.md]: The point is that an index is free to use whatever terms are in a readers's head and is not restricted to what are used in the product "paperwork".

**Pull request merged**